apps/admin-gui/src/app/shared/components/dialogs/edit-attribute-definition-dialog/edit-attribute-definition-dialog.component.scss

```diff
@@ -0,0 +1,46 @@
+.unique-toggle-label {
+  font-size: 1.125rem;
+}
+
+.width-450 {
+  width: 450px !important;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+:host {
+  //display: block;
+  overflow: hidden;
+}
+
+.parent {
+  //height: 100%;
+  //width: 200%;
+  display: flex;
+  div { flex: 1; }
+}
+
+//.mat-dialog-height-transition {
+//  transition: height 0.5s !important;
+//}
+
+//.animate-show {
+//  height: 0;
+//  background: white;
+//  overflow: hidden;
+//  transition: all 0.5s;
+//}
+//
+//.animate-show.ng-hide-add, .animate-show.ng-hide-remove {
+//  transition: all 0.5s;
+//}
+//
+//.animate-show:not(.ng-hide) {
+//  height: 60px;
+//}
```

Comment:
Same here.

Reply:
This is code for animation, I will remove when I finish the animation